### ISSUE TITLE
Feature: Switch on dark mode

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#467]
Most of these bots would run in headless mode anyways but it would be nice to programmatically switch on dark mode for the more esoteric users out there.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.45](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.44...1.9.45) (2020-05-23)

- Release 1.9.45 [`8e3f418`](https://github.com/open-wa/wa-automate-nodejs/commit/8e3f4188ced0ec126f47938e520971f9299c3505)
- insiders feature: implemented `darkMode` #467 [`adf2835`](https://github.com/open-wa/wa-automate-nodejs/commit/adf2835743117b205411b41becf619b7d1551ba4)
- insiders feature: implemented `setChatBackgroundColourHex` #470 [`2b169f7`](https://github.com/open-wa/wa-automate-nodejs/commit/2b169f79b512fd2ec91d910b398f55115eb9d10e)

**Comment by smashah** [comment]
https://open-wa.github.io/wa-automate-nodejs/classes/client.html#darkmode
